releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,50 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Updated support for AAD. System Assigned identity previously required; config store name, and the Contributor role. Now it requires; config store config store name, [client id][client_id], and the App Configuration Data Reader role.
+
+```properties
+spring.cloud.azure.appconfiguration.managed-identity.client-id=[client id]
+spring.cloud.azure.appconfiguration.stores[0].name=[config store name]
+```
+
+* Updated support for AAD. User Assigned identity previously required; config store name, client id, object id, and the Contributor role. Now it required; config store name, client id, and the App Configuration Data Reader role.
+* Added the following Provider to allow users to provide any [Azure TokenCredentials][token_credentials] to the provider for authenticating with App Configuration & Key Vault. When a `TokenCredentialProvider` `@Bean` is made it will be used for authentication.
+
+**Note:** Null is valid value to be returned if only one of the authentication methods needs to be used.
```

Comment:
I think we shouldn't fall back to the managed identity if the user code returns null. This is too magic.  If a user is already implementing a `TokenCredentialProvider`, it's simple enough for them to return a `ManagedIdentityCredential`.

Reply:
@Zhenlan The idea is that they could use the Provider for Key Vault, but a connection string for App Configuration